extensions/binding/org.eclipse.smarthome.binding.mqttbroker/src/main/java/org/eclipse/smarthome/binding/mqttbroker/MqttBrokerBindingConstants.java

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.binding.mqttbroker;
+
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+
+/**
+ * The {@link MqttBrokerBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author David Graeff - Initial contribution
+ */
+public class MqttBrokerBindingConstants {
+
+    private static final String BINDING_ID = "mqttbroker";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_TEXTUAL = new ThingTypeUID(BINDING_ID, "textualBroker");
+    public static final ThingTypeUID THING_TYPE_BROKER = new ThingTypeUID(BINDING_ID, "broker");
+
+    // General parameters
+
+    public static final String PARAM_BROKER_url = "url";
```

Comment:
Only comment once: I don't think this mix of upper and lower case is something we used ever. static final constants should be in uppercase only.